### PR TITLE
docs: update contribution guides [skip-bc]

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-3114

### Changes included:

some feedbacks reported by @e-krebs for the contribution guides, also some alignment on the names scripts->docs

### CODE REVIEW

File: website/docs/setup-repository.md [hunk 1]
```diff
@@ -18,28 +20,26 @@ nvm use && yarn
...
-yarn docker:setup
-```
+We use multiple docker images to simplify the build, and reuse them on the CI.
+There is one base image containing node, java, scala and kotlin, located in [scripts/docker/Dockerfile](https://github.com/algolia/api-clients-automation/blob/main/scripts/docker/Dockerfile)
 
-### Docker
+:::caution
 
-#### Build
+The swift images takes a really long time to build (~5 minutes) because of swift-format, but it's only needed when you want to format swift.
```
Comment (millotp) [inline website/docs/setup-repository.md:28]: I think this is not the case anymore, the slowest is the base one but it's always required

File: website/docs/setup-repository.md [hunk 2]
```diff
@@ -18,28 +20,26 @@ nvm use && yarn
 
 ## Mounting the docker images
 
-```bash
-yarn docker:setup
-```
+We use multiple docker images to simplify the build, and reuse them on the CI.
```
Comment (millotp) [inline website/docs/setup-repository.md:23]:
```suggestion
We use multiple docker images to simplify the build.
```

File: website/docs/CLI/build-commands.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+### Build playground for specific language
+
+```bash
+apic build playground javascript recommend
+```
+
+### Build many snippets for specific language
+
+```bash
+apic build snippets php insights recommend search
```
Comment (millotp) [inline website/docs/CLI/build-commands.md:106]: php build is a noop still 😢 
`phpstan` is too hard

**Review by millotp — COMMENTED** [review]
this is so nice

File: website/docs/setup-repository.md [hunk 4]
```diff
@@ -18,28 +20,26 @@ nvm use && yarn
...
 ```
 
-:::caution
+## Tooling 
 
-The swift images takes a really long time to build (~5 minutes) because of swift-format, but it's only needed when you want to format swift.
+:::info
 
-:::
+This step is optional, using `yarn cli ...` should work perfectly as well.
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply website/docs/setup-repository.md:40]: yup but a bit intrusive so... that's fine I guess

File: website/docs/CLI/build-commands.md (hunk 3 above)
Comment (shortcuts) [reply website/docs/CLI/build-commands.md:106]: I can put python instead :trollface:

File: website/docs/setup-repository.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (millotp) [reply website/docs/setup-repository.md:33]: `apic` is mostly about the auto-completion rather than the speed, I think this is the main advantage

**Pull request merged**